quantum-template.tex

```diff
@@ -181,8 +181,8 @@ \subsection{References and footnotes}
 
 \section{Plots}
 \label{sec:plots}
-Quantum provides a \href{https://jupyter.org/}{Jupyter notebook} based on the widely used \href{https://matplotlib.org/}{matplotlib} library that greatly simplifies the creation of plots that integrate seamlessly with the \texttt{quantumarticle} document class. This is intended as a service to the authors, is \textit{not} mandatory, and currently in beta stage. You can download the \href{https://raw.githubusercontent.com/quantum-journal/quantum-journal/master/quantum-plots.ipynb}{quantum-plots.ipynb} notebook and accompanying \href{https://raw.githubusercontent.com/quantum-journal/quantum-journal/master/quantum-plots.mplstyle}{quantum-plots.mplstyle} file from the \href{https://github.com/quantum-journal/quantum-journal}{quantumarticle GitHub repository}. You only need to specify the font size and paper format that were passed as options to \texttt{quantumarticle} to get plots with fitting font sizes and dimensions.
-We strongly encourage authors to use the vector based PDF format for plots.
+We strongly encourage authors to use the vector based PDF format for plots. The Python package \href{https://pypi.org/project/rsmf/}{rsmf} (``right-size my figures'') together with the widely used plotting library \href{https://matplotlib.org/}{matplotlib} can be used to create plots that integrate seamlessly with the \texttt{quantumarticle} document class. In the Quantum \href{https://github.com/quantum-journal/quantum-journal/}{GitHub repository}, you find the script \texttt{example-plot.py} that shows how these tools were used to create Figure~\ref{fig:figure1}.
```

Comment:
My only request: Can we make "example-plot.py" a direct download link to the file from the master branch? Probably best via https://raw.githubusercontent.com/quantum-journal/quantum-journal/master/...

Reply:
Done :)